src/Language/PureScript/Interactive.hs

```diff
@@ -217,12 +226,15 @@ handleImport
   => ImportedModule
   -> m ()
 handleImport im = do
-   st <- gets (updateImportedModules (im :))
+   st <- gets (updateImportedModules (renew im))
    let m = createTemporaryModuleForImports st
    e <- liftIO . runMake $ rebuild (map snd (psciLoadedExterns st)) m
    case e of
      Left errs -> printErrors errs
      Right _  -> put st
+   where
+   renew :: ImportedModule -> [ImportedModule] -> [ImportedModule]
```

Comment:
It's to prevent import duplication. Previously, when a module was imported twice, it was listed in the imported modules twice. Maybe the name renew is somewhat confusing?